### PR TITLE
Old wall building!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## Description
Using cloth on wooden barricades will now make indoor walls yay!
Using leather(I have no idea for any other easy to get mat that can be used a lot) on wooden barricades to make house wlls
## Motivation and Context
I hate seeing indoor walls being mixed with wooden ones, just annoying to have to remake all the indoor nice looking walls into the cabin feel in a wasteland 
## How Has This Been Tested?
nope...

## Screenshots (if appropriate):
sad
## Changelog (necessary)
:cl: Lilly Trilby
add: Hey handy man, and carpenters alike! Thanks to some brave souls throwing everything at the wall and seeing what sticks, they have re-learned the old ways to make indoor walls like back before the bombs dropped as well as those nice house walls! Rejoy as the way to make indoor walls is cloth, not to shocking, and the old house walls is... Leather??? 
/:cl:

### CODE REVIEW

**Comment by Laus4Deus** [comment]
Without testing, we can't merge 😞

File: code/modules/fallout/obj/structures/obstacle.dm [hunk 1]
```diff
@@ -67,18 +67,34 @@
 	material = WOOD
 
 /obj/structure/barricade/wooden/attackby(obj/item/weapon/I, mob/living/user, params)
-	if(!istype(src, /obj/structure/barricade/wooden/planks) && !istype(src, /obj/structure/barricade/wooden/crude) && istype(I, /obj/item/stack/sheet/mineral/wood))
+	if(!istype(src, /obj/structure/barricade/wooden/planks) && !istype(src, /obj/structure/barricade/wooden/crude) && istype(I, /obj/item/stack/sheet/))
 		if(isfloorturf(loc) || isplatingturf(loc))
```
Comment (Laus4Deus) [inline code/modules/fallout/obj/structures/obstacle.dm:71]: I can't find the floor & plating turfs. I assume plating is only for metal floor, but floorturf?

Comment (Trilbyspaceclone) [reply code/modules/fallout/obj/structures/obstacle.dm:71]: floor and plating are not the same

Comment (Laus4Deus) [reply code/modules/fallout/obj/structures/obstacle.dm:71]: Oh, totally!
The comment doesn't really have much to do with the pr itself, sorry.

**Pull request merged**